test/test_metric/test_metrics.py

```diff
@@ -295,6 +295,63 @@ def test_threshold_scorer_multilabel(self):
         score = scorer(y_true, y_pred)
         self.assertAlmostEqual(score, -1.0)
 
+    def test_threshold_scorer_multi_class(self):
+        y_true = np.array([[0, 1], [1, 0], [1, 0], [0, 1]])
+        y_pred = np.array([[0.2, 0.8], [0.75, 0.25], [0.91, 0.09], [0.34, 0.66]])
+
+        scorer = autosklearn.metrics._ThresholdScorer(
+            "roc_auc_ovo_macro", sklearn.metrics.roc_auc_score, 1, 0, 1, {}
+        )
+
+        score = scorer(y_true, y_pred)
+        self.assertAlmostEqual(score, 1.0)
+
+        y_pred = np.array([[0.1, 0.9], [0.15, 0.85], [0.2, 0.8], [0.25, 0.75]])
+        score = scorer(y_true, y_pred)
+        self.assertAlmostEqual(score, 0.5)
+
+        y_pred = np.array([[0.75, 0.25], [0.8, 0.2], [0.85, 0.15], [0.9, 0.1]])
+        score = scorer(y_true, y_pred)
+        self.assertAlmostEqual(score, 0.5)
+
+        scorer = autosklearn.metrics._ThresholdScorer(
+            "roc_auc_ovo_macro",
+            sklearn.metrics.roc_auc_score,
+            1,
+            0,
+            1,
+            {"average": "weighted"},
+        )
+        y_pred = np.array([[0.6, 0.4], [0.6, 0.4], [0.8, 0.2], [0.8, 0.2]])
+        score = scorer(y_true, y_pred)
+        self.assertAlmostEqual(score, 0.5)
+
+    def test_threshold_scorer_multilabel_multiclass(self):
```

Comment:
To the best of my knowledge, there is no metric in scikit-learn that can handle this yet, so I would advise to remove this. In case there is any, could you please let us know which one it is?